docs/self-managed/identity/user-guide/additional-features/incorporate-applications.md

```diff
@@ -5,11 +5,7 @@ sidebar_label: "Incorporate applications"
 description: "Use Identity to create an application and assign a permission to an application."
 ---
 
-In this guide we will show you how to use Identity to create an application and assign a permission to an application.
-
-:::tip Want to learn more about applications?
```

Comment:
I removed this and used an in-line link in the sentence above since we have a caution admonition right below it.